src/datumaro/components/merge/extractor_merger.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from typing import Optional, Sequence, TypeVar
+
+from datumaro.components.contexts.importer import _ImportFail
+from datumaro.components.dataset_base import DatasetBase, SubsetBase
+
+T = TypeVar("T")
+
+
+def check_identicalness(seq: Sequence[T], raise_error_on_empty: bool = True) -> Optional[T]:
```

Comment:
Why don't you include thin function in `ExtractorMerger`? This is expected to be utilized for other mergers from `ExtractorMerger`.

Reply:
It is used in `COCOTaskMergedBase` but there is no inheritance between `COCOTaskMergedBase` and `ExtractorMerger`.